### ISSUE TITLE
Crashed when pressing 'Picture' for second time

### ISSUE THREAD

**Issue by thiyagab** [EzequielAdrianM/Camera2Vision#8]
The sample worked flawlessly except for one small bug,

When I am trying to multiple pictures, and while pressing 'picture'  it crashed in the second attempt,

The error was,
```
java.lang.IllegalStateException: maxImages (2) has already been acquired, call #close before acquiring more.
at android.media.ImageReader.acquireNextImage(ImageReader.java:501)
at com.example.ezequiel.camera2.others.Camera2Source$PictureDoneCallback.onImageAvailable(Camera2Source.java:1446)
```

**Comment by thiyagab** [comment]
First I tried to close the imagereader didnt work, then closing the acquired image solved the problem

**Comment by StephenAkers** [comment]
This issue is fixed in PR #15
